code/game/machinery/iv_drip.dm

```diff
@@ -179,7 +179,7 @@
 	src.update_icon()
 
 /obj/machinery/iv_drip/attack_ai(mob/living/user)
-	attack_hand(user)
+	return
```

Comment:
undocumented change?

Reply:
I think I meant to delete those lines entirely, since its inherited from /obj/machinery